CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ option( EXIV2_ENABLE_WEBREADY         "Build webready support into library"
 option( EXIV2_ENABLE_CURL             "USE Libcurl for HttpIo (WEBREADY)"                     OFF )
 option( EXIV2_ENABLE_BMFF             "Build with BMFF support"                               ON  )
 
-option( EXIV2_BUILD_SAMPLES           "Build sample applications"                             ON  )
+option( EXIV2_BUILD_SAMPLES           "Build sample applications"                             OFF )
```

Comment:
I had to disable that because the workflow `.github/workflows/cifuzz.yml` was failing to compile the project with clang14 (due to the issue with the Jzon class + C++17).